src/screens/Home/Home.tsx

```diff
@@ -37,6 +38,70 @@ interface HomeProps {
 
 function Home(props: HomeProps) {
   const { classes, lastUpdated, isOnline, currentSite } = props;
+  useEffect(() => {
+    proccessData();
+  }, []);
+
+  const [outStandingPayments, setOutstandingPayments] = useState<number>();
+  const [toCharge, setToCharges] = useState<number>();
+  const [customerDetails, setCustomerDetails] = useState<number>();
+  const [unpaidReports, setUnpaidReports] = useState<number>();
```

Comment:
I don't think this has to be a stateful component. We can calculate the data on render and show it without using state